src/server/controllers/match.controller.js

```diff
@@ -33,14 +33,15 @@ export function requestMentoring(req, res, next) {
       })
       .then(mentor => {
         if (mentor) {
-          sendEmail(receiver, mailStrings.REQUEST_SUBJECT, mailStrings.REQUEST_HTML, matchData.content);
+          // TODO: Confirm method whether send mail or send in-app message.
+          // sendEmail(receiver, mailStrings.REQUEST_SUBJECT, mailStrings.REQUEST_HTML, matchData.message);
```

Comment:
Because we didn't confirm about this. We can even use both of them later.
But if this isn't neat, I'll remove it.
What do you think?